lib/temporal/connection/converter/codec/base.rb

```diff
@@ -0,0 +1,33 @@
+module Temporal
+  module Connection
+    module Converter
+      module Codec
+        class Base
+          def encodes(payloads)
+            return nil if payloads.nil?
+
+            Temporalio::Api::Common::V1::Payloads.new(
+              payloads: payloads.payloads.map(&method(:encode))
+            )
+          end
+
+          def decodes(payloads)
+            return nil if payloads.nil?
+
+            Temporalio::Api::Common::V1::Payloads.new(
+              payloads: payloads.payloads.map(&method(:decode))
+            )
+          end
```

Comment:
Would these be overridden by someone extending this class? Why not move these into `payloads.rb`?

Reply:
they can be, I just was following the same pattern as in `payloads.rb` which seems to be define the same thing (`from_payloads` and `to_payloads` with logic then `from_payload` and `to_payload` are empty and required)

You probably are going to only override `encode` or `decode` but you have the option to override all for if the use case needs it.